### ISSUE TITLE
Can I use gtoolbar with tcltk?

### ISSUE THREAD

**Issue by zhibisora** [jverzani/gWidgets2#106]
```R
require(tcltk)
require(gWidgets2)
require(gWidgets2tcltk)
```
then
```R
win <- gwindow('R',visible=TRUE)
my_Open <-  gaction(label="Open", icon="open",handler=function(h,...){
    # ...
})
my_list <- list(
  open = my_Open,
  sep = list(separator = TRUE))
gtb <- gtoolbar(toolbar.list = my_list, container = win)
```
that will cause an error
```R
Error in UseMethod(".gtoolbar") : 
  no applicable method for '.gtoolbar' applied to an object of class "c('guiWidgetsToolkittcltk', 'guiWidgetsToolkit')"
```

So, can I use gtoolbar with tcltk?

**Comment by jverzani** [comment]
Sorry, this is definitely a bug. I am getting around to fixing this up, so hopefully can identify and close soon.

**Comment by jverzani** [comment]
This should be closed in release 1.0-7. Thanks for the bug report.
